src/vm/prestub.cpp

```diff
@@ -245,7 +246,18 @@ PCODE MethodDesc::PrepareInitialCode()
 {
     STANDARD_VM_CONTRACT;
     PrepareCodeConfig config(NativeCodeVersion(this), TRUE, TRUE);
-    return PrepareCode(&config);
+    PCODE pCode = PrepareCode(&config);
+
+#if defined(FEATURE_GDBJIT) && defined(FEATURE_PAL) && !defined(CROSSGEN_COMPILE)
+    if (!tls_isSymReaderInProgress)
```

Comment:
Could you please move the tls_isSymReaderInProgress definition and usage to the gdbjit.cpp? It seems there is no need to have it exposed out of it. I know it was outside of it before, but I have not noticed it when I was reviewing the code in the past.